frontend/public/components/dashboards-page/overview-dashboard/details-card.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { connect } from 'react-redux';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../dashboard/dashboard-card';
+import { DetailsBody, DetailItem } from '../../dashboard/details-card';
+import { withDashboardResources, DashboardItemProps, FirehoseResource } from '../with-dashboard-resources';
+import { InfrastructureModel } from '../../../models';
+import { referenceForModel } from '../../../module/k8s';
+import { FLAGS } from '../../../const';
+import { featureReducerName, flagPending } from '../../../reducers/features';
+import { RootState } from '../../../redux';
+
+const OPENSHIFT_VERSION_QUERY = 'openshift_build_info{job="apiserver"}';
+
+const getClusterName = (infrastructure: object): string => {
+  const apiServerURL = _.get(infrastructure, 'status.apiServerURL');
```

Comment:
Note that you don't need to use the `Infrastructure` resource to get this. It's in `window.SERVER_FLAGS`

Reply:
That yields different result - `window.SERVER_FLAGS.kubeAPIServerURL` is `https://10.37.160.45:6443` but `status.apiServerURL` has translated the ip to hostname ie `test1.tt.testing`

Reply:
The `kubeAPIServerURL` value is set from the `Infrastructure` resource. Is this a development environment?

Reply:
yes, it is. We did setup the env using openshift-installer and Libvirt povider.

Reply:
The operator sets the same value from the `Infrastructure` resource:

https://github.com/openshift/console-operator/blob/2d7462da12c406fe31ef9d51207499cc0a2716c1/pkg/console/subresource/configmap/configmap.go#L53

It should match exactly unless you're running a local bridge binary. If not, it's a bug.

Reply:
I was using local bridge, my bad. Thanks for pointing that out. Using `kubeAPIServerURL` now.